freespace2/freespace.cpp

```diff
@@ -2993,10 +2989,29 @@ void apply_view_shake(matrix *eye_orient)
 			tangles.p += get_shake(0.07f * cut_pct * sn_shudder, -1, 0);
 			tangles.h += get_shake(0.07f * cut_pct * sn_shudder, -1, 0);
 		}
+
+		if (Game_shudder_everywhere) {
+			do_global_shudder = true;
+		}
+	}
+
+	if (do_global_shudder) {	
+		// Make eye shake due to shuddering
+		if (Game_shudder_perpetual) {
+			tangles.p += get_shake(Game_shudder_intensity * 0.005f, -1, 0);
+			tangles.h += get_shake(Game_shudder_intensity * 0.005f, -1, 0);
+		} else if (Game_shudder_time.isValid()) {
+			if (timestamp_elapsed(Game_shudder_time)) {
+				Game_shudder_time = TIMESTAMP::invalid();
+			} else {
+				tangles.p += get_shake(Game_shudder_intensity * 0.005f, timestamp_until(Game_shudder_time), Game_shudder_total);
+				tangles.h += get_shake(Game_shudder_intensity * 0.005f, timestamp_until(Game_shudder_time), Game_shudder_total);
+			}
+		}
 	}
 
 	// maybe bail
-	if (tangles.p == 0.0f && tangles.h == 0.0f && tangles.b == 0.0f)
```

Comment:
I take it changing to the now default fl_near_zero threshold does not visually change things?

Reply:
Not that I can see.